creator-node/src/config.js

```diff
@@ -458,7 +458,7 @@ const config = convict({
     doc: 'number of state monitoring jobs that can run in each interval (0 to pause queue)',
     format: 'nat',
     env: 'stateMonitoringQueueRateLimitJobsPerInterval',
-    default: 0
+    default: 3
```

Comment:
`config.js` is generally used to set prod defaults - can you update that, also should open PR for stage/prod config values in
https://github.com/AudiusProject/audius-docker-compose/tree/main/creator-node and https://github.com/AudiusProject/audius-k8s/tree/master/helm/charts/creator

Reply:
the intent here is to set prod defaults -- I'll open PRs for the other repos and try to merge them around the same time

Reply:
i misunderstood this, code is fine